.vitepress/config.mts

```diff
@@ -1,28 +1,206 @@
+                ]
+              }
+            ]
+          },
+          {
+            text: 'Other Stuff',
+            items: [
+              { text: 'Team', link: '/en/team' },
+              { text: 'FAQ', link: '/en/faq' },
+              { text: 'Support', link: '/en/support' },
+              { text: 'Feedback', link: '/en/feedback' },
+              { text: 'Contribution', link: '/en/contribution' },
+              { text: 'Roadmap', link: '/en/roadmap' },
+              { text: 'Changelog', link: '/en/changelog' },
+              { text: 'Tech Stack', link: '/en/tech-stack' }
+            ]
+          }
+        ]
+      }
+    },
+    de: {
+      label: 'Deutsch',
+      lang: 'de', // optional, will be added  as `lang` attribute on `html` tag
+      link: '/de/', // default /de/ -- shows on navbar translations menu, can be external
+      themeConfig: {
+        search: {
+          provider: 'local',
+          options: {
+            locales: {
+              de: {
+                translations: {
+                  button: {
+                    buttonText: 'Suchen'
+                  },
+                  modal: {
+                    noResultsText: 'Keine Ergebnisse für',
+                    resetButtonTitle: 'Suche zurücksetzen',
+                    displayDetails: 'Detaillierte Liste anzeigen',
+                    footer: {
+                      selectText: 'um auszuwählen',
+                      navigateText: 'um zu navigieren',
+                      closeText: 'zum schließen'
+                    }
+                  }
+                }
+              }
+            }
+          }
+        },
+        darkModeSwitchTitle: 'Wechsle zum Dark Theme',
+        lightModeSwitchTitle: 'Wechsle zum Light Theme',
+        lastUpdated: {
+          text: 'Aktualisiert am',
+          formatOptions: {
+            dateStyle: 'full',
+            timeStyle: 'medium'
+          }
+        },
+        editLink: {
+          pattern: 'https://github.com/arma-events/docs/edit/main/src/:path',
+          text: 'Bearbeite diese Seite auf GitHub'
+        },
+        docFooter: {
+          prev: 'vorherige Seite',
+          next: 'nächste Seite'
+        },
+        nav: [
+          { text: 'FAQ', link: '/de/faq' },
+          { text: 'Support', link: '/de/support' },
+          { text: 'Feedback', link: '/de/feedback' },
+          { text: 'Mitwirkung', link: '/de/contribution' }
+        ],
+        sidebar: [
+          {
+            text: 'Artikel',
+            items: [
+              { text: 'Überblick', link: '/de/overview' },
+              { text: 'Benutzer', link: '/de/users' },
+              { text: 'Communities', link: '/de/communities' },
+              {
+                text: 'Events', 
+                link: '/de/events', 
+                items: [
+                  { text: 'Markdown', link: '/de/markdown' },
+                  { text: 'Slotsliste', link: '/de/slotlist' }
```

Comment:
```suggestion
                  { text: 'Slotliste', link: '/de/slotlist' }
```

oder

```suggestion
                  { text: 'Slotlist', link: '/de/slotlist' }
```

Reply:
Upsi. Da macht man einmal ein `cSpell: disable` und schon haut es einem in die Fresse. :sob: